js.go

```diff
@@ -952,6 +952,10 @@ func (opt subOptFn) configureSubscribe(opts *subOpts) error {
 }
 
 // Subscribe will create a subscription to the appropriate stream and consumer.
+// By default, it will automatically create an JetStream consumer in the server in case
+// one cannot be found. Unless the `Durable` option is passed, the consumer will be
+// ephemeral and automatically cleaned up by the server if the client that created
+// the consumer goes away.
```

Comment:
Correct, but then the library is still trying to delete on Unsusbcribe(), so we will have to address that.